src/WebJobs.Script/Description/Compilation/CompilerErrorException.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Scripting;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    [Serializable]
+    public sealed class CompilerErrorException : Exception
```

Comment:
nit: `CompilerErrorException` and `CompilationErrorException` sound very similar -- is there another name we can give this? Even `CompilerException` sounds different enough to me...